Sources/FoundationEssentials/URL/URL.swift

```diff
@@ -1688,41 +1688,47 @@ extension URL {
     /// Checks if a file path is absolute and standardizes the inputted file path on Windows
     /// Assumes the path only contains `/` as the path separator
     internal static func isAbsolute(standardizing filePath: inout String) -> Bool {
+        if filePath.utf8.first == ._slash {
+            return true
+        }
         #if os(Windows)
```

Comment:
Refactored the Windows part and moved up the `.first == ._slash` check, which is valid for Windows and non-Windows platforms.